CHANGELOG.md

```diff
@@ -1,7 +1,28 @@
 # Change Log
+
 All notable changes to this project will be documented in this file.
 See the [change log guidelines](http://keepachangelog.com/) for information on how to structure the file.
 
+## [5.1.0]
+
+### Added
+
+- Simplify the setup process with automated creation of deegree configurations and capabilities for WFS/WMS services.
+- Introduced support for the MS Access JDBC driver, expanding connectivity options for your database needs.
```

Comment:
I would classify that as a **change**, since nothing new was implemented, just the related dependency added.

I would propose:
"Include support for MS Access"